README.rst

```diff
@@ -101,6 +124,13 @@ project. If you have any questions, or to execute our corporate CLA, required
 if your contribution is on behalf of a company, please drop us an email at
 opensource@newrelic.com.
 
+A note about vulnerabilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As noted in our `security policy <https://github.com/newrelic/.github/blob/master/SECURITY.md>`_, New Relic is committed to the privacy and security of our customers and their data. We believe that providing coordinated disclosure by security researchers and engaging with the security community are important means to achieve our security goals.
```

Comment:
This security link should link to the security tab rather than the `.github` repo since the security tab is a bit more permanent.
```suggestion
As noted in our `security policy <https://github.com/newrelic/newrelic-python-agent/security/policy>`_, New Relic is committed to the privacy and security of our customers and their data. We believe that providing coordinated disclosure by security researchers and engaging with the security community are important means to achieve our security goals.
```